### ISSUE TITLE
Question about WISDEM Installation

### ISSUE THREAD

**Issue by DingZL1207** [WISDEM/WISDEM#467]
When I tried to install WISDEM, I encountered the following issue at the beginning. My computer already has 64-bit Anaconda installed. Could you please guide me on how to resolve this problem? Thank you!
![image](https://github.com/WISDEM/WISDEM/assets/141538354/1f2d47b3-55e9-4929-b739-e6fa13879ece)

**Comment by DingZL1207** [comment]
’SpecNotFound: Invalid name 'python=3.10', try the format: user/package‘

**Comment by gbarter** [comment]
Try the same command, just remove `python=3.10`

**Comment by DingZL1207** [comment]
> Try the same command, just remove `python=3.10`

thank you very much!I did it!

**Comment by DingZL1207** [comment]
> Try the same command, just remove `python=3.10`

When I started 'First Steps in WISDEM', I encountered the following issues even though my environment was already configured. What should I do in this situation?
![image](https://github.com/WISDEM/WISDEM/assets/141538354/c2933604-35af-4f00-95c8-6738c70947f2)

**Comment by DingZL1207** [comment]
> > Try the same command, just remove `python=3.10`
> 
> When I started 'First Steps in WISDEM', I encountered the following issues even though my environment was already configured. What should I do in this situation? ![image](https://user-images.githubusercontent.com/141538354/276064547-c2933604-35af-4f00-95c8-6738c70947f2.png)

![image](https://github.com/WISDEM/WISDEM/assets/141538354/8fc0d7d4-e821-42d3-a0a2-3154d7e5c78f)

**Comment by herminiomiguel** [comment]
When I tried to install WISDEM, I encountered the following issue at the beginningm, even though I removed "python=3.10".  Could you please guide me on how to resolve this problem? Thank you!

![image](https://github.com/WISDEM/WISDEM/assets/151202522/c57cd7fa-5615-4f3c-bca3-cbbf59ee709f)


I tryed to change for mamba as suggested on the WISDEM homepage, but the same problem happens.

![image](https://github.com/WISDEM/WISDEM/assets/151202522/1a3696e5-665c-4038-919f-b89feea66cd1)

![image](https://github.com/WISDEM/WISDEM/assets/151202522/45ad1640-1358-4b1b-88d9-bce25620ac9d)

![image](https://github.com/WISDEM/WISDEM/assets/151202522/c4928b68-ee0f-4567-8341-17f44a9e62f9)

![image](https://github.com/WISDEM/WISDEM/assets/151202522/b9ff5737-7c3b-4212-8554-dd16abad43c3)

![image](https://github.com/WISDEM/WISDEM/assets/151202522/1a4bd829-a0e3-4efd-a847-bb734d2fc01e)

**Comment by gbarter** [comment]
This isn't a WISDEM problem here.  You've entered a proxy server in your conda configuration that isn't valid: `id:pw@address:port`.  That is an example of how to enter user id, password, proxy address, and proxy port.  I would check with your institution IT for the actual settings, as this is site-specific and not something we can help with.

**Comment by herminiomiguel** [comment]
Thank you for your tip, Barter! The installation finished successfully after I logged into admin account. And sorry, it is my first time working with Python.

After the installation, I ran the test_all.py. and all the 12 figures were shown to me, but the test didn't reach 100% (see below the last two lines presented). Several time later (close to 6 hours) I checked the task manager and the python/terminal no longer worked, thus I applied ctrl+c to stop it. The aforementioned procedure was carried out 2 times with the same result. In this case, everithing is ok?

..\wisdem\test\test_drivetrainse\test_layout.py .........                                                                                            [ 21%]
..\wisdem\test\test_examples\test_examples.py

Take the opportunity to ask another question, this time related to "7-Generator Model Examples". After to run "pmsg_outer.py" I could observe the efficiency values below. I didn't find information about the units used, but I supposed it is in "%". If so, indeed, they are extremely low, although it's just an example. I tested changing some settings such as costs or rated power and the same pattern continues.

converter_efficiency    |4.4069257|            None     converter_efficiency
  transformer_efficiency  |4.45901931|           None     transformer_efficiency
  generator_efficiency    |4.22641729|           None     generator_efficiency
  
  Thank you in advance!
